### ISSUE TITLE
Updated firmware not driving LED

### ISSUE THREAD

**Issue by sstaszko** [Aharoni-Lab/Miniscope-DAQ-Cypress-firmware#6]
I updated the firmware on my v3.2 DAQ without issue and also updated to the newest version of the software. Whenever I connect my scopes to the DAQ everything lights up as it should, but neither the old or updated version of the miniscope control software will drive the LED power to the scope. When I connect my scopes to a second DAQ/software I haven't updated, everything is working as normal. When I revert back to the older firmware, the DAQ/LED works as it should.

**Comment by ximion** [comment]
Did you use the prebuilt firmware binaries in `Built_Firmware/` or build the code from source?

**Comment by sstaszko** [comment]
I used the prebuilt firmware binary. I actually tried both thinking maybe I was wrong about my DAQ version, but only the v3.2 was successfully loaded via Cypress.

**Comment by daharoni** [comment]
@sstaszko are you using a V3 or V4 Miniscope?

**Comment by sstaszko** [comment]
I am using the V3 scope currently, I was updating in anticipation of V4!

**Comment by daharoni** [comment]
Hmm very strange as I have spoken to a few other people using the newest firmware and software with the V3 without issue.

Are you able to stably connect to the V3 and adjust framerate and gain through the new software GUI sliders?

**Comment by sstaszko** [comment]
Yes, it is connected stably and will let me alter things through the GUI. I can "change" the LED power on the scopes, but it doesn't actually change the power coming from the LED, so without imaging I'm not sure if my changes of frame rate are real if that makes sense.

**Comment by daharoni** [comment]
Sorry for the delay in getting back to you. You can see in the software if changing the FPS actually worked. In the top middle of the video window it shows the instantaneous and average FPS of the video stream. This number should change when you select a different FPS.
